src/tests/ie_test_utils/common_test_utils/common_utils.hpp

```diff
@@ -19,6 +19,8 @@
 
 namespace CommonTestUtils {
 
+using SizeVector = std::vector<size_t>;
```

Comment:
```suggestion
```

Reply:
Please do not introduce new test structures.

Reply:
done